### PR TITLE
[integration_test] Add integration_test_tizen package

### PR DESCRIPTION
**Pull request by swift-kim**
Copied from the development branch for initial release.

### CODE REVIEW

File: packages/integration_test/example/tizen/NuGet.Config
Comment (bbrto21) [inline packages/integration_test/example/tizen/NuGet.Config:6]: ```localpackages``` are added, please check again

Comment (swift-kim) [reply packages/integration_test/example/tizen/NuGet.Config:6]: What a mistake!

**Review by bbrto21 — APPROVED** [review]
LGTM

**Comment by swift-kim** [comment]
Please do not merge the PRs yet. I'll merge when we're ready to publish on pub.dev.

**Pull request merged**
